### PR TITLE
WIP: title fallback using contract name and tokenID

### PR DESCRIPTION
**Pull request by dblodorn**
@iainnash this would be a nice feature so we "always" have a title available for the Token.
I did notice that when using betaIndexer the contract title & symbol are not available, so perhaps we include this in  nft-hooks query, or simply surface the contract name and symbol in the nft object.

Screen Shot 2021-12-02 at 3 06 43 PM

Screen Shot 2021-12-02 at 2 55 24 PM

Screen Shot 2021-12-02 at 2 55 17 PM

### CODE REVIEW

**Comment by dblodorn** [comment]
@iainnash FWIW - this is an interesting metadata contrast between betaIndexer and external data source:

Screen Shot 2021-12-17 at 4 38 23 PM

Screen Shot 2021-12-17 at 4 27 35 PM

File: src/nft-full/CollectionTag.tsx [hunk 1]
```diff
@@ -7,11 +7,13 @@ import { Orb } from "../components/Orb";
...
   const {
-    nft: { data },
+    nft: { data }
   } = useContext(NFTDataContext);
- 
-  const { getStyles } = useMediaContext();
   
+  const { getStyles } = useMediaContext();
+
+  console.log(data)
```
Comment (iainnash) [inline src/nft-full/CollectionTag.tsx:15]: nit rm console.log

File: src/nft-full/CollectionTag.tsx [hunk 2]
```diff
@@ -21,12 +23,14 @@ export const CollectionTag = () => {
         rel="noreferrer"
       >
         <div {...getStyles("collectionTagIcon")}>
-          {/* @ts-ignore */ data && "openseaInfo" in data
-            ? <img src={data.openseaInfo.asset_contract.image_url} alt={data.openseaInfo.asset_contract.name}/>
+          {data && "openseaInfo" in data && data.openseaInfo.asset_contract.image_url !== null
```
Comment (iainnash) [inline src/nft-full/CollectionTag.tsx:26]: there should be data.nft.contract.image now from opensea

File: src/nft-full/MediaInfo.tsx [hunk 3]
```diff
@@ -18,8 +18,10 @@ export const MediaInfo = ({ a11yIdPrefix, className }: MediaInfoProps) => {
 
   const getContent = () => {
     if (metadata && data) {
+      console.log(metadata)
```
Comment (iainnash) [inline src/nft-full/MediaInfo.tsx:21]: nit console.log

**Review by iainnash — CHANGES_REQUESTED** [review]

**Comment by iainnash** [comment]
This is stale, will close for now.

**Pull request closed without merging**